pkgs/development/tools/buf/default.nix

```diff
@@ -26,11 +27,12 @@ buildGoModule rec {
     ./skip_test_requiring_dotgit.patch
   ];
 
-  nativeBuildInputs = [ protobuf ];
+  nativeBuildInputs = [ protobuf installShellFiles ];
   # Required for TestGitCloner
   checkInputs = [ git ];
 
   ldflags = [ "-s" "-w" ];
+  tags = [ "-trimpath" ];
```

Comment:
I believe `-trimpath` is the default for `buildGoModule`, and also this isn't a valid build tag.